.jshintrc

```diff
@@ -44,7 +44,6 @@
     "expr"          : false,       // Suppresses warnings about the use of expressions where normally you would expect to see assignments or function calls.
     "funcscope"     : false,       // Suppresses warnings about declaring variables inside of control structures while accessing them later from the outside.
     "gcl"           : false,       // Makes JSHint compatible with Google Closure Compiler.
-    "globalstrict"  : false,       // Suppresses warnings about the use of global strict mode.
```

Comment:
Note to reviewers: this is an out-of-date option which is [getting deprecated](http://jshint.com/docs/options/#globalstrict). By removing the option, we still don't tolerate global strict mode (no semantic change), but protect ourselves against future build failures.